docs/coordinates/galactocentric.rst

```diff
@@ -179,8 +179,11 @@ globally set the frame defaults at runtime. For example, the default parameter
 values can be seen by initializing the `~astropy.coordinates.Galactocentric`
 frame with no arguments::
 
+    >>> import warnings
     >>> from astropy.coordinates import Galactocentric
-    >>> Galactocentric()
+    >>> with warnings.catch_warnings():
+    ...     warnings.simplefilter('ignore')  # Ignore deprecation warning
+    ...     Galactocentric()
```

Comment:
Can't we let the warning be shown and include it in the expected output?

Reply:
Will that silence `pytest` warnings?

Reply:
That's a good question! If it doesn't, what about including the warning here anyway and then using doctest SKIP?

Reply:
Skipping code is better than two lines to catch warning? I am not convinced.

Reply:
How about we skip this particular test, and add a block near the bottom of the file with a short example of how to catch the warning, and do this down there?